web/docs/language/features.md

```diff
@@ -839,6 +839,7 @@ The quickest way to get started is by using the following API generated by Wasp:
 - Signup and Login forms at `@wasp/auth/forms/Signup` and `@wasp/auth/forms/Login` routes
 - `logout` function
 - `useAuth()` React hook
+It should be noted that the Signup form automatically logs in the user after signup.
```

Comment:
```suggestion
**NOTE:** If the signup is successful, the Signup form will automatically log in the user.
```
Just to follow the style we use in the rest of the docs :)

Reply:
Excellent.  :+1:

Reply:
Wow, I just saw that I can commit the suggestion from the UI.  :exploding_head:

Reply:
Yeah, pretty cool. I've only learned about it recently too.